.github/workflows/rebase.yml

```diff
@@ -0,0 +1,26 @@
+name: Automatic Rebase
+on:
+  issue_comment:
+    types: [created]
+jobs:  
+  rebase:
+    name: Rebase
+    runs-on: ubuntu-latest
+    if: >-
+      github.event.issue.pull_request != '' && 
+      (
+        contains(github.event.comment.body, '/rebase') || 
+        contains(github.event.comment.body, '/autosquash')
```

Comment:
I think this won't quite implement #10904, `autosquash` performs `git rebase --autosquash`, which doesn't squash _all_ commits.